packages/server/tests/acceptance/equivalence.spec.ts

```diff
@@ -0,0 +1,884 @@
+  });
+
+  // EQV-008 - should be successfull - OK
+  // Skipped as test is not ready
+  it.skip('direct CALL to address over 1000 without amount (payable contract)', async function () {
+    const { contractExecuteTimestamp } = await servicesClient.executeContractCall(
+      equivalenceDestructContractId,
+      NON_EXISTING_FUNCTION,
+      EMPTY_FUNCTION_PARAMS,
+    );
+
+    const record = await getResultByEntityIdAndTxTimestamp(equivalenceDestructContractId, contractExecuteTimestamp);
+    validateContractCall(record, equivalenceDestructContractId, SUCCESS, STATUS_SUCCESS);
+  });
+
+  // EQV-009 - OK -  should be unsuccessfull
+  it('should execute direct call to address over 1000 with amount - case non-payable contract ', async function () {
+    const args = [estimatePrecompileContractAddress, NON_EXISTING_FUNCTION, EMPTY_FUNCTION_PARAMS, 500_000, 100];
+    await testRejection(CONTRACT_REVERT_EXECUTED, servicesNode.executeContractCallWithAmount, true, servicesNode, args);
+  });
+
+  // Skipped as test is not ready
+  it.skip('should execute direct call to address over 1000 with amount', async function () {
+    let contractCallResult;
+    try {
+      contractCallResult = await servicesClient.executeContractCallWithAmount(
+        ETH_PRECOMPILE_0x1001,
+        NON_EXISTING_FUNCTION,
+        EMPTY_FUNCTION_PARAMS,
+        1_000_000,
+        100,
+      );
+    } catch (e) {
+      const contractActions = await getContractActions(getTransactionIdFromException(e));
+      assert.fail(`${e.message}\ncontact actions:\n${JSON.stringify(contractActions, null, 2)}`);
+    }
+
+    const record = await getResultByEntityIdAndTxTimestamp(
+      ETH_PRECOMPILE_0x1001,
+      contractCallResult.contractExecuteTimestamp,
+    );
+    validateContractCall(record, ETH_PRECOMPILE_0x1001, SUCCESS, CONTRACT_EXECUTION_EXCEPTION);
+  });
+
+  // EQV-13 This should be successful due to hollow account creation
+  // Skipped as test is not ready
+  it.skip('should execute direct call to address over 1000 with amount - case hollow acconut', async function () {
+    const hollowAccount = ethers.Wallet.createRandom();
+    // convert evm address to hedera address
+    const hollowAcAddress = hollowAccount.address.toString();
+    let contractCallResult;
+
+    try {
+      contractCallResult = await servicesClient.executeContractCallWithAmount(
+        hollowAcAddress,
+        NON_EXISTING_FUNCTION,
+        EMPTY_FUNCTION_PARAMS,
+        1_000_000,
+        100,
+      );
+    } catch (e) {
+      const contractActions = await getContractActions(getTransactionIdFromException(e));
+      assert.fail(`${e.message}\ncontact actions:\n${JSON.stringify(contractActions, null, 2)}`);
+    }
+
+    const record = await getResultByEntityIdAndTxTimestamp(
+      ETH_PRECOMPILE_0x1001,
+      contractCallResult.contractExecuteTimestamp,
+    );
+    validateContractCall(record, ETH_PRECOMPILE_0x1001, SUCCESS, CONTRACT_EXECUTION_EXCEPTION);
+  });
+
+  [
```

Comment:
Extracted above the test body as 'basicTests' (commonTests). Regarding exporting them in separate file it would reduce visibility and I would advice against it except a case where those exact same tests will be reusable elsewhere.